File: jw/week67/PowerOfThree.java
class Solution {
    public boolean isPowerOfThree(int n) {
        if(n == 0) return false;
        boolean rs = true;
        if(n >= 1) {
            while(n > 1) {
                if(n%3 != 0 ) return false;
                n/=3;
            }
        } else  {
            return false;
        }
        return rs;
    }
}

File: sj/67week/Power of Three.js
/**
 * @param {number} n
 * @return {boolean}
 */
var isPowerOfThree = function(n) {
    
   while (n > 1) {
       
        if (n % 3 === 0){
             n /= 3;
        }
        else {
             break;
        }
    }
    
    return n === 1;
    
};
